### DISCUSSION TITLE
Sleep mode excessively draining battery

### DISCUSSION THREAD

**Discussion by OmegaMoog** [eduardofilo/RG350_adam_image#102] [General]
RG280v and I'm on Adam 1.2. When pressing power button and Y to initate sleep mode I seem to be experiencing a lot of power drain. The battery % falls about 20% every 5 hours. When in sleep mode the power LED remains lit.

**Comment by eduardofilo** [comment]
This is the normal behavior of the sleep mode of OD Beta, i.e. the LED stays on. 20% for 5 hours seems reasonable. I guess the sleep function is better suited for shorter game pauses. If you are going to have the console suspended for more than 5 hours, it would be advisable to turn it off.

**Comment by OmegaMoog** [comment]
Thank you for confirming that this behaviour is normal.
